### ISSUE TITLE
allow linkRel for individual link op

### ISSUE THREAD

**Issue by zeitwidrig** [nozer/quill-delta-to-html#59]
Same as with linkTarget it should be possible to define rel values for each individual link op. For instance you have one paragraph with two links: one internal, one external.
```javascript
{
        'attributes': {
            'link': '#',
            'target': '_blank',
            'rel': 'nofollow noopener'
          },
          'insert': 'external link'
        },
        {
          'attributes': {
            'link': '#'
          },
          'insert': 'internal link'
        }
}
```

### Expected output
```html
<a href="#" target="_blank" rel="nofollow noopener">external link</a><a href="#">internal link</a>
```
### Actual output
```html
<a href="#" target="_blank">external link</a><a href="#">internal link</a>
```

### Notes
global linkRel option is not applicable since "rel=nofollow" shouldn't be used for *every* link including internal links.

**Comment by nozer** [comment]
You are absolutely right. I just pushed a fix with tag `v0.10.8` that work as you expected.

**Comment by zeitwidrig** [comment]
Works great, thanks.
